test/e2e/hello-world-encrypted-SIM.json

```diff
@@ -3,7 +3,7 @@
     "name": "hello-world-encrypted-container"
   },
   "image":{
-    "image": "docker.io/mylinen/hello-world-encrypted:samplekbc2"
+    "image": "docker.io/xynnn007/helloworld:sample_kbc_enc"
```

Comment:
As part of https://github.com/kata-containers/tests/issues/5521, my goal is to be able to get rid of personal images and have all of the test images in the same place.  Now that the tests for this PR are green, can we get the images to moved to the same place where the rest of the test coco images?

Reply:
Agreed. Now enclave-cc uses two test encrypted images
- decrypted by sample-kbc in SIM: actually `sample-kbc` is not used anymore for kata-cc
- decrypted by eaa-kbc (replace with cc-kbc in future): I suggest to replace with [ccv0-ssh](https://hub.docker.com/layers/xynnn007/ccv0-ssh/encrypted/images/sha256-ac0ed3364d54120a1025c74f555b21cb378d4d0f62a398c5ce3e1b89fa4ca637?context=repo) (which I have not test yet, though), which will be moved to ghcr.io in HW

Reply:
@Xynnn007 Do you think it makes sense to continue using `sample-kbc` based image for SIM? (btw, we have this `hello` image used in operator e2e too: https://github.com/confidential-containers/operator/blob/88a0a60b47bf76dcc9a37452a53063df0917a37c/tests/e2e/enclave-cc-pod-sim.yaml#L9)

Reply:
I suggest not. We can use `offline-fs-kbc` instead to test key brokering without remote attestation -- which means that we should get rid of `sample-kbc`. How do you feel?

Reply:
Does it mean the decryption keys must be copied in the occlum instance?

Reply:
no, we do not. we just need to set up a file `/etc/offline-fs-kbc-keys.json` on the ci machine like [this](https://github.com/confidential-containers/image-rs/blob/main/test_data/offline-fs-kbc/aa-offline_fs_kbc-keys.json). Then enclave-agent with `attestation_agent/offline_fs_kbc` enabled can look up for keys from that file.

But seems that the code freeze is near, we can just use this sample kbc encrypted image, and start to change next release?

I think it is not a big problem